clients/csrf/default_csrf_token_fetcher.go

```diff
@@ -24,38 +21,33 @@ func NewDefaultCsrfTokenFetcher(transport *Transport) *DefaultCsrfTokenFetcher {
 	return &DefaultCsrfTokenFetcher{transport: transport}
 }
 
-func (c *DefaultCsrfTokenFetcher) FetchCsrfToken(url string, currentRequest *http.Request) (*csrf_parameters.CsrfRequestHeader, error) {
-
+func (c *DefaultCsrfTokenFetcher) FetchCsrfToken(url, authToken string) (csrf_parameters.CsrfParams, error) {
 	fetchTokenRequest, err := http.NewRequest(http.MethodGet, url, nil)
 	if err != nil {
-		return nil, err
+		return csrf_parameters.CsrfParams{}, err
 	}
 	fetchTokenRequest.Header.Set(XCsrfToken, CsrfTokenHeaderFetchValue)
```

Comment:
It's an old code, but I do not think this "Fetch" value is required on the backend